### DISCUSSION TITLE
API SSL issue

### DISCUSSION THREAD

**Discussion by fanclubfm** [supabase/supabase#2130] [Questions] [answered]
Any ideas how to fix this issue? Started happening recently.  It appears as if SSL is not configured or has issues.  
https://xriebkmejfviuvnvndfq.supabase.co/rest/v1/

Dashboard cannot connect to Tables or to API section. 

Client gives this error specifically "failed, reason: write EPROTO 4425162240:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:332"

Thanks

**Comment by inian** [comment, marked as answer]
Hi,

Yes, this is due to Google flagging a subdomain of ours. The issue was resolved within an hour but some local routers cached this information from the Safe Browsing dataset from Google.

There is more context in this Github issue - https://github.com/supabase/supabase/issues/2096#issuecomment-869438174

Thanks

**Reply by fanclubfm** [reply]
Thanks.  All good now.  Appreciate a quick response.

**Reply by fanclubfm** [reply]
Just FYI this started happening again tonight on both of my projects (pro and free).  Thx.
